packages/generator/src/subtask2/my-func.ts

```diff
@@ -0,0 +1,89 @@
+import ts,{ factory } from 'typescript';
+import { TableColumn } from './column';
+import { createJsxElement } from './component';
+
+
+
+
+
+
+function createAst(
+    code: string,
+    scriptTarget = ts.ScriptTarget.ESNext,
+    scriptKind = ts.ScriptKind.TSX,
+    filePath = `/ts-ast-viewer.tsx`
+  ) {
+    return ts.createSourceFile(
+      filePath,
+      code,
+      scriptTarget,
+      true,
+      scriptKind
+    )
+  }
+
+const importCheck = (SourceFile:ts.SourceFile|ts.Node) => {
+      return <boolean>ts.forEachChild(SourceFile, node => {
+        if(ts.isImportDeclaration(node)){
+          let temparr = node.importClause?.namedBindings?.getChildren()[1].getChildren().map((n) => n.getText()=='TableColumn'?true:false) 
+          if (temparr && temparr.indexOf(true) > -1){
+            return true
+          }
+        }
+      })
+}  
+
+export function createNewColumn(file: string, newColumnName: string): ts.SourceFile {
+    const sourceFile = createAst(file);
+    let isImported:boolean = importCheck(sourceFile)
```

Comment:
importCheck() function now returns true or false, but I don't want it to run for every loop, so created an input